k8s/plugin/README.md

```diff
@@ -28,31 +28,31 @@ The plugin needs to be able to connect to the REST server in order to make the a
 1. Get Volumes
 ```
 ❯ kubectl mayastor get volumes
```

Comment:
I think we should have namespaces on these sample commands. It is unintuitive to have the default namespace set to 'mayastor'.

Reply:
I've updated README to detail what the default namespace is.